show/muxcable.py

```diff
@@ -189,7 +189,7 @@ def status(port, json_output):
             port_status_dict["MUX_CABLE"] = {}
             for namespace in namespaces:
                 asic_id = multi_asic.get_asic_index_from_namespace(namespace)
-                for key in port_table_keys[asic_id]:
+                for key in sorted(port_table_keys[asic_id], key=lambda x: int(x.split("|")[1].replace("Ethernet", ""))):
```

Comment:
Should we use the `natsort` module to sort the port names using natural sort?

Reply:
Good idea, will update